package.json

```diff
@@ -37,7 +37,7 @@
     "nyc": "^10.1.2"
   },
   "bin": {
-    "debundle": "src/index.js"
+    "debundle": "src/cmdline.js"
```

Comment:
I'd prefer if there wasn't a distinction between `cmdline.js`/`index.js` (all in one file), and that file acted as both the command line client as well as a library. See below comments for clarification.